Dockerfile

```diff
@@ -1,19 +1,34 @@
-FROM ubuntu
+FROM ubuntu:18.04
 
-# System requirements
-RUN apt-get update && apt-get install -y \
-  build-essential \
-  curl \
-  python3-pip \
-  swig \
-  && rm -rf /var/lib/apt/lists/*
+WORKDIR /auto-sklearn
+
+# Copy the checkout autosklearn version for installation
+ADD . /auto-sklearn/
+
+# install linux packages
+RUN apt-get update
+
+# Set the locale
+RUN apt-get -y install locales
+RUN touch /usr/share/locale/locale.alias
```

Comment:
Could you briefly explain what these lines are needed for?

Reply:
These lines are needed for the UTF encoding error during lazy import installation. Please let me know if you are aware of a better WA to overcome UnicodeDecodeErrors while pip installing this module